flavors/standard-Exasol-8-cuda-ml/flavor_base/language_deps/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM {{udfclient_deps}}
+
+RUN mkdir -p /build_info/packages/language_deps
+COPY language_deps/packages/ /build_info/packages/language_deps
+
+ENV CONDA_OVERRIDE_CUDA=12.6
```

Comment:
Would be nice if we can externalize this

Reply:
Created https://github.com/exasol/script-languages-release/issues/1035